playbooks/common/openshift-cluster/node_docker_ca.yml

```diff
@@ -0,0 +1,128 @@
+---
+- name: Configure CA certificate for secure registry
```

Comment:
@detiber We only need to know the service IP and route here so that should work.